utils.groovy

```diff
@@ -959,4 +959,48 @@ def build_and_push_image(params = [:]) {
     return manifest_digest
 }
 
+def get_arch_image_digest(from, arch) {
+    return shwrapCapture("""\
+        skopeo inspect --raw docker://${from} | \
+        jq -r '.manifests[] | select(.platform.architecture == "${arch}" and .platform.os == "linux") | .digest'
+    """.stripIndent())
```

Comment:
You'd think `--override-arch` would make this easier, but nope ([related comment](https://github.com/coreos/fedora-coreos-pipeline/pull/1137#discussion_r2071874431)).